### DISCUSSION TITLE
List operation question

### DISCUSSION THREAD

**Discussion by alexsol22** [Laravel-Backpack/community-forum#396] [Q&A (Help)] [answered]
is it possible based on the filter to combine columns of the dataTable? 
for Example: If the filter is active on a particular record it must show me only some columns if instead on another other columns

**Comment by pxpm** [comment, marked as answer]
Hey @alexsol22 

It could be possible to do that, but you need to overwrite/create a new filter that instead of triggering a table redraw with the new url, triggers a full page reload.

You can then intercept on your controller in the `request()`, check what/if filters are active and `addColumn()` or `removeColumn()` depending on that.

Cheers

**Reply by alexsol22** [reply]
Thanks, I'll try as you suggested
